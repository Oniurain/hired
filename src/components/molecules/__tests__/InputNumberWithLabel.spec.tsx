import * as React from 'react'
import { mount } from 'enzyme'

import { InputNumberWithLabel } from '../'

describe('InputNumberWithLabel', () => {
  it('should render', () => {
    const subject = mount(
      <InputNumberWithLabel
        id="rate"
        label="Rate"
        placeholder="Enter the current rate for the project"
        unit="EUR"
        changeHandler={() => {}}
      />
    )

    expect(subject).toMatchSnapshot()
  })
})
