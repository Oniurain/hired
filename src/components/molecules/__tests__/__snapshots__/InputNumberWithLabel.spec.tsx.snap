// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputNumberWithLabel should render 1`] = `
<InputTextWithLabel
  changeHandler={[Function]}
  id="rate"
  label="Rate"
  placeholder="Enter the current rate for the project"
  unit="EUR"
>
  <Styled(BaseRow)>
    <BaseRow
      className="dHOzQD"
    >
      <div
        className="dHOzQD"
      >
        <styled.label
          htmlFor="rate"
        >
          <label
            className="fPhlMm"
            htmlFor="rate"
          >
            Rate
          </label>
        </styled.label>
        <styled.input
          id="rate"
          onChange={[Function]}
          placeholder="Enter the current rate for the project"
          type="number"
        >
          <input
            className="gGJqti"
            id="rate"
            onChange={[Function]}
            placeholder="Enter the current rate for the project"
            type="number"
          />
        </styled.input>
        <styled.div>
          <div
            className="dlMZJJ"
          >
            EUR
          </div>
        </styled.div>
      </div>
    </BaseRow>
  </Styled(BaseRow)>
</InputTextWithLabel>
`;
